Code/Tools/AssetProcessor/native/AssetManager/FileStateCache.cpp

```diff
@@ -7,9 +7,10 @@
  */
 
 #include "FileStateCache.h"
-#include "native/utilities/assetUtils.h"
-#include <AssetProcessor_Traits_Platform.h>
 
+#include <AssetProcessor_Traits_Windows.h>
```

Comment:
it seems odd to see a platform specific include like AssetProcessor_Traits_Windows.h, right?

Reply:
I thought that too.  So I removed it, then the compile failed and pointed out why.  There's a define check to see if we're on a case sensitive file system.  On case-insensitive filesystems the path key is lowercased before storing it

Reply:
I thought there was a AssetProcessor_Traits.h that included the proper PAL header file. No?

Reply:
AssetProcessor_Traits_Platform.h

Reply:
Ah, ended up including the wrong thing when I put it back.  Fixed!